processor/sumologicprocessor/metadata.yaml

```
@@ -0,0 +1,7 @@
+type: sumologic
+
+status:
+  class: processor
+  stability:
+    beta: [traces, metrics, logs]
+  distributions: [core, contrib, splunk, observiq, sumo, aws, redhat]
```

```suggestion
  distributions: [contrib, sumo]
```
I'm guessing this will be the case at first?